src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -56,22 +56,22 @@ void EquationInputArea::OnEquationsPropertyChanged()
     }
 }
 
-void EquationInputArea::AddEquationButton_Click(Object ^ sender, RoutedEventArgs ^ e)
-{
-    AddNewEquation();
-}
-
 void EquationInputArea::AddNewEquation()
 {
     auto eq = ref new EquationViewModel(ref new Equation());
+    eq->IsLastItem = true;
+
+    if (Equations->Size > 0)
+    {
+        Equations->GetAt(Equations->Size - 1)->IsLastItem = false;
+    }
 
     m_lastLineColorIndex = (m_lastLineColorIndex + 1) % AvailableColors->Size;
 
     eq->LineColor = AvailableColors->GetAt(m_lastLineColorIndex);
     eq->IsLineEnabled = true;
     eq->FunctionLabelIndex = ++m_lastFunctionLabelIndex;
     Equations->Append(eq);
-    EquationInputList->ScrollIntoView(eq);
```

Comment:
I could see some benefit in leaving this so that the user can see that there is a ghost equation at the bottom of the scroll view when scrolling starts to occur.

Reply:
The scrolling cause the issue we spoke about yesterday (itemContainer recycled with the wrong value).


![image](https://user-images.githubusercontent.com/1226538/69700570-a1494700-109f-11ea-8880-7cc2c12c8384.png)

I will bring it back once EquationTextBox will be binded to the VM